CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Added
 
-- nothing added
+- add process autoregister
```

Comment:
Description of feature added is not long enough

Reply:
more details are added